src/test/java/it/EndpointTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ *******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *   2018-06-19 - Jon Hawkes / IBM Corp
+ *      Initial code
+ *
+ *******************************************************************************/
+
+package it;
+
+import static org.junit.Assert.assertTrue;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.Response;
+
+public class EndpointTest {
+
+    public void testEndpoint(String endpoint, int expectedStatus, String expectedOutput) {
+        String port = System.getProperty("liberty.test.port");
```

Comment:
Maybe the property should be renamed to `test.port` and it defaults to 8080 but is overridden in Liberty Profile to another value if needed?

Reply:
yes. i'll do that.